.pre-commit-config.yaml

```diff
@@ -21,7 +21,13 @@ repos:
     rev: '3.8.1'  # pick a git hash / tag to point to
     hooks:
     -   id: flake8
+        additional_dependencies:
+          - flake8-rst-docstrings
 -   repo: https://github.com/psf/black
     rev: 19.10b0
     hooks:
     - id: black
+-   repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.5.1
+    hooks:
+      - id: rst-backticks
```

Comment:
FYI you may also like this hook that Anthony has: https://github.com/asottile/setup-cfg-fmt#as-a-pre-commit-hook